src/utils/document.js

```diff
@@ -90,6 +90,7 @@ export function Document({ Html, Head, Body, children, state }) {
         <meta name="twitter:description" content={meta.twitter.description || meta.description} />
         <meta name="twitter:image" content={SITE_ROOT + meta.twitter.image} />
         <link rel="shortcut icon" href={meta.favicon} type="image/x-icon" />
+        <meta name="zoom-domain-verification" content="ZOOM_verify_Col76fg4SlWOfEajLOxodQ"></meta>
```

Comment:
```suggestion
        <meta name="zoom-domain-verification" content="ZOOM_verify_Col76fg4SlWOfEajLOxodQ" />
```

I think you are good with a self-closing tag since there's nothing happening between, but otherwise lgtm!